sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseEditAuthorization.java

```diff
@@ -0,0 +1,77 @@
+package de.symeda.sormas.backend.caze;
```

Comment:
@HolgerReiseVSys What do you think about this solution to move the authorization logic to it's own class? I think it's a good idea, but I want to make sure I'm not missing anything important.